internal/gmm/handler.go

```diff
@@ -2159,6 +2160,34 @@ func HandleRegistrationComplete(ue *context.AmfUe, accessType models.AccessType,
 		})
 	}
 
+	// Send NITZ information to UE
```

Comment:
Please make this part a common api since it is similar to the one in AmPolicyControlUpdateNotifyUpdateProcedure.

Reply:
Did you mean that implements an API for sending **configuration update command**?

Reply:
Yes, I think the api may take ConfigurationUpdateCommandFlags as input parameter and does BuildConfigurationUpdateCommand, judges whether to start timer, and does SendConfigurationUpdateCommand in it. 
Also, registration complete -> ConfigurationUpdateCommand may not be restricted 3GPP access only, so maybe we need to pass accessType as parameter or take au.AccessType into consideration for that api.